### PR TITLE
Test(MetaList): adds test cases for MetaList Component

### PR DESCRIPTION
**Pull request by anuradha9712**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/atoms/metaList/Meta.tsx [hunk 1]
```diff
@@ -13,9 +13,16 @@ export const Meta = (props: MetaProps) => {
...
+    <span data-test="DesignSystem-MetaList--Meta" className={'Meta'}>
+      {icon && (
+        <Icon
+          data-test="DesignSystem-MetaList--Icon"
+          name={icon}
+          appearance={iconAppearance}
+          className={'Meta-icon'}
+        />
+      )}
+      <Text data-test="DesignSystem-MetaList--Label" appearance={labelAppearance}>{label}</Text>
```
Comment (riyalohia) [inline core/components/atoms/metaList/Meta.tsx]: data-test should be `DesignSystem-MetaList--MetaLabel`

File: core/components/atoms/metaList/__tests__/MetaList.test.tsx [hunk 2]
```diff
@@ -0,0 +1,249 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { testHelper, filterUndefined, valueHelper, testMessageHelper } from '@/utils/testHelper';
+import MetaList, { MetaListProps as Props } from '../MetaList';
```
Comment (riyalohia) [inline core/components/atoms/metaList/__tests__/MetaList.test.tsx]: `import { MetaList } from '@/index';`
`import { MetaListProps as Props } from '@/index.type';`

File: core/components/atoms/metaList/__tests__/MetaList.test.tsx [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+      expect(tree).toMatchSnapshot();
+    });
+  };
+
+  testHelper(mapper, testFunc);
+});
+
+describe('MetaList component prop:seperator options', () => {
+  const mapper = {
+    seperator: valueHelper(booleanValues, { iterate: true }),
```
Comment (riyalohia) [inline core/components/atoms/metaList/__tests__/MetaList.test.tsx]: `seperator: valueHelper(booleanValues, { iterate: true, required: true })`

File: core/components/atoms/metaList/__tests__/MetaList.test.tsx [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+
+  const testFunc = (props: Record<string, any>): void => {
+    const attr = filterUndefined(props) as Props;
+
+    it(testMessageHelper(attr), () => {
+      const tree = render(
+        <MetaList
+          {...attr}
+          list={[{ icon: 'assessment', label: 'Meta data' }]}
+          seperator={true}
```
Comment (riyalohia) [inline core/components/atoms/metaList/__tests__/MetaList.test.tsx]: No need to pass seperator prop

File: core/components/atoms/metaList/__tests__/MetaList.test.tsx [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+    expect(getByTestId('DesignSystem-MetaList--Meta').tagName).toMatch('SPAN');
+  });
+});
+
+describe('MetaList Component with overwrite class', () => {
+  it('overwrite MetaList class', () => {
+    const { getByTestId } = render(<MetaList list={[{ icon: 'assessment', label: 'Meta data' }]} className="MetaListClass" />);
+    expect(getByTestId('DesignSystem-MetaList')).toHaveClass('MetaListClass');
+  });
+});
```
[the remark this answers is not in the document]
Comment (anuradha9712) [reply core/components/atoms/metaList/__tests__/MetaList.test.tsx:255]: Updated!!

**Review by riyalohia — APPROVED** [review]

**Pull request merged**